djangocms_form_builder/settings.py

```diff
@@ -20,7 +20,15 @@
 framework = getattr(django_settings, "DJANGOCMS_FRONTEND_FRAMEWORK", "bootstrap5")
 theme = getattr(django_settings, "DJANGOCMS_FRONTEND_THEME", "djangocms_frontend")
 
-DEFAULT_SPACER_SIZE_CHOICES = (("mb-3", "Default"),)
```

Comment:
I'd rather ask you to set this in your settings. It will first try to take `DJANGO_FORM_BUILDER_SPACER_CHOICES` then take django CMS frontend's settings if available.